src/platform_impl/linux/x11/monitor.rs

```diff
@@ -21,32 +21,20 @@ impl XConnection {
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
 pub struct VideoModeHandle {
     pub(crate) current: bool,
-    pub(crate) size: (u32, u32),
+    pub(crate) size: PhysicalSize<u32>,
     pub(crate) bit_depth: Option<NonZeroU16>,
     pub(crate) refresh_rate_millihertz: Option<NonZeroU32>,
```

Comment:
Would it be nicer to store the `VideoMode` struct here to avoid repetition, or is that making the access patterns elsewhere more ugly?